packages/walletconnect-web3-subprovider/src/index.js

```diff
@@ -88,7 +98,7 @@ export default class WalletConnectSubprovider extends Subprovider {
   }
   sendAsync(payload, callback) {
     const next = () => {
-      const sendAsync = this.engine.sendAsync.bind(this)
+      const sendAsync = this._walletconnect.sendAsync.bind(this)
```

Comment:
Are you still getting an issue with this issue? I've successfully triggered JSON-RPC requests on mobile using the `walletconnect-web3-provider` package using the README instructions. If this issue is related with Walleth specifically, I've contacted @ligi and he said that hasn't updated to the latest spec.

Reply:
Sorry. A leftover from #38

Reply:
The above engine vs _walletconnect I will check.